src/generator/writeClients.ts

```diff
@@ -89,8 +89,16 @@ function writeClients(dictionary: IReferenceDirectory[]) {
     interfacesTypesImports.forEach((type) => {
       clientFile += `import ${type} from '../interfaces/types/${type}Types';\n`;
     });
-
+    clientFile += `import Transport from '../base/transport'\n`
     clientFile += `\nclass ${fileName}Client {\n`;
+    clientFile += `   
+    private transport: Transport;
+    constructor(transport: Transport) {
+      if (!transport) {
+        throw Error('Transport class needs to be provided.');
+      }
+      this.transport = transport;
```

Comment:
And a final empty line here before assigning `this.transport` :smile:

Reply:
Done